tests/baselines/reference/allowImportingTsExtensions(moduleresolution=classic).errors.txt

```diff
@@ -0,0 +1,32 @@
+/c.ts(1,16): error TS2792: Cannot find module './thisfiledoesnotexist.ts'. Did you mean to set the 'moduleResolution' option to 'nodenext', or to add aliases to the 'paths' option?
+
+
+==== /ts.ts (0 errors) ====
+    export {};
+    
+==== /tsx.tsx (0 errors) ====
+    export {};
+    
+==== /dts.d.ts (0 errors) ====
+    export {};
+    
+==== /b.ts (0 errors) ====
+    import {} from "./ts.js";
+    import {} from "./ts.ts";
+    import type {} from "./ts.d.ts";
+    
+    import {} from "./tsx.js";
+    import {} from "./tsx.jsx";
+    import {} from "./tsx.ts";
+    import {} from "./tsx.tsx";
+    import type {} from "./tsx.d.ts";
+    
+    import {} from "./dts.js";
+    import {} from "./dts.ts";
+    import type {} from "./dts.d.ts";
+    
+==== /c.ts (1 errors) ====
+    import {} from "./thisfiledoesnotexist.ts";
+                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS2792: Cannot find module './thisfiledoesnotexist.ts'. Did you mean to set the 'moduleResolution' option to 'nodenext', or to add aliases to the 'paths' option?

```

Comment:
why is this correct error message when the file doesnt exist ?

Reply:
Oh, I actually didn’t look close enough at this one. This is weird; I think it’s only supposed to show up for non-relative names.

Reply:
Hm, this message just always comes up in `classic`, probably because `classic` is terrible and nobody should use it. It doesn’t make a ton of sense for relative imports, but honestly if someone is on `classic` it probably _is_ a mistake, so I think we should leave it.